### ISSUE TITLE
Slot Configuration instructions in the Manual | please add some information

### ISSUE THREAD

**Issue by lansingthomas** [atlas-engineer/nyxt#3182]
Hi Nyxt contributors :D
This thing has been driving me nuts and an improvement might be helpful for Luka 's `CommonSettings/external-text-editor configuration` overhaul. So it gets to live here now.

**Is your feature request related to a problem? Please describe.**
1. I want to use an external text editor (and some other features)
2. I understand that to do this I must configure a SLOT
3. I am a n00b
4. I try the `Nyxt - Manual` page
5. I search C-f for "slot"
6. I find "Slot Configuration"
7. The page tells me what slots are, and why I might want to change one.
8. The page DOES NOT tell me HOW to change a slot.

**Describe the solution you'd like**
I expect that Luka will want something like this for his work in Common Settings and it's probably best to write it as a team. 

**Describe alternatives you've considered**
looking it up online by myself (never).

**Additional context**
#### I tried the example under Slot Configuration and found some issues with the example:
1. following the example verbatim changes the page, so instructions are lost (maybe add a step for opening a new nyxt window)
2. I got to step 5 "insert the string asfdghjkl" and didn't know why I was doing this, or if it worked.
   - then I see an error message "Warning: Type mismatch for HINTS-ALPHABET: got SYMBOL, expected STRING."
   - I got lucky and guessed that parentheses would fix this because Lisp... and there is no success message!
   - also why"asfdghjkl" instead of "asdfghjkl"?

Thank you for your continued hard work on this awesome project.

**Comment by jmercouris** [comment]
Yes, but it should be smart enough to do type checking and prompt the user until they input the correct value

**Comment by lansingthomas** [comment]
sorry I was not very clear -- I am hoping for a description in the manual for HOW to change a slot.  I sort of threw on a few other things at the end.

**Comment by jmercouris** [comment]
Perhaps we can include a link to the relevant section in the manual?

**Comment by jmercouris** [comment]
I mean next to the slot description when someone does `describe-slot` or `describe-class`?

**Comment by lansingthomas** [comment]
Lets provide a slightly easier example --

updated text:
---

# Slot Configuration

Slots store values that can be configured for customizing Nyxt. For instance, the slot ZOOM-RATIO-DEFAULT can be set to zoom pages to a comfortable setting.

Here is how it's done:

1. Execute the command `describe-slot (f1 s)`;
2. Type to select ZOOM-RATIO-DEFAULT from the WEB-BUFFER class;
3. Press the Configure button, and;
4. Insert the float value 2.0
5. after restarting Nyxt, every page will be zoomed in 2x.

Feel free to try this with commonly customized slots such as:
- DEFAUL-NEW-BUFFER-URL, to change your homepage for new buffers.
- DEFAULT-MODES, to tell Nyxt which modes to use for new buffers.
